plugins/modem-manager/fu-mm-device.c

```diff
@@ -442,6 +458,34 @@ typedef struct {
 	gsize		 total_bytes;
 } FuMmArchiveIterateCtx;
 
+static gboolean
+should_be_active (const gchar *version,
```

Comment:
This should have a prefix like `fu_mm` to avoid potential re-use elsewhere in fwupd clashing

Reply:
Done